setup/alpine34/resources/apk.repositories.j2

```diff
@@ -0,0 +1,17 @@
+http://dl-1.alpinelinux.org/alpine/v3.4/main
+http://dl-2.alpinelinux.org/alpine/v3.4/main
+http://dl-3.alpinelinux.org/alpine/v3.4/main
+http://dl-4.alpinelinux.org/alpine/v3.4/main
+http://dl-5.alpinelinux.org/alpine/v3.4/main
+
+http://dl-1.alpinelinux.org/alpine/edge/testing
+http://dl-2.alpinelinux.org/alpine/edge/testing
+http://dl-3.alpinelinux.org/alpine/edge/testing
+http://dl-4.alpinelinux.org/alpine/edge/testing
+http://dl-5.alpinelinux.org/alpine/edge/testing
```

Comment:
Is community or testing available for v3.4? If so, lets use that instead. Also, what do we need from `testing`? I recall pulling java from the `community` repo.